contracts/modules/governance/NXMaster.sol

```diff
@@ -217,90 +240,37 @@ contract NXMaster is INXMMaster, Governed {
     return mr.checkRole(_add, uint(IMemberRoles.Role.Member));
   }
 
-  ///@dev Gets the number of emergency pause has been toggled.
-  function getEmergencyPausedLength() public view returns (uint len) {
-    len = emergencyPaused.length;
-  }
-
-  ///@dev Gets last emergency pause details.
-  function getLastEmergencyPause() public view returns (bool _pause, uint _time, bytes4 _by) {
-    _pause = false;
-    _time = 0;
-    _by = "";
-    uint len = getEmergencyPausedLength();
-    if (len > 0) {
-      len = len.sub(1);
-      _pause = emergencyPaused[len].pause;
-      _time = emergencyPaused[len].time;
-      _by = emergencyPaused[len].by;
-    }
-  }
-
-  /// @dev Gets latest version name and address
-  /// @return contractsName Latest version's contract names
-  /// @return contractsAddress Latest version's contract addresses
-  function getVersionData()
+  /// @dev Gets current contract codes and their addresses
+  /// @return contractCodes
+  /// @return contractAddresses
+  function getInternalContracts()
```

Comment:
External?

Reply:
i'll leave it public for now, its only minor